modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -716,6 +793,14 @@ protected JValue eventToJSON(Event event, Boolean withAcl, Boolean withMetadata,
     fields.add(f("identifier", v(event.getIdentifier(), BLANK)));
     fields.add(f("location", v(event.getLocation(), BLANK)));
     fields.add(f("presenter", arr($(event.getPresenters()).map(Functions.stringToJValue))));
+    if (!requestedVersion.isSmallerThan(VERSION_1_1_0)) {
+      fields.add(f("language", v(event.getLanguage(), BLANK)));
+      fields.add(f("rightsholder", v(event.getRights(), BLANK)));
+      fields.add(f("license", v(event.getLicense(), BLANK)));
+      fields.add(f("is_part_of", v(event.getSeriesId(), BLANK)));
```

Comment:
@krinnewitz Ooops, seems I did an error in the specification in MH-12756: I think the name should be isPartOf to be consistent with the /metadata request. I will double-check all field names and report back later (no need for action on your side)

Reply:
From my point of view: Yes. See the fieldnames.pdf in https://opencast.jira.com/browse/MH-12808 to get an idea of the current state of the field names.